index.html

```diff
@@ -0,0 +1,209 @@
+
+          </figure>
+
+        </div>
+          
+        <div class="main">
+
+          <div class="container">     
+         
+            <div class="left-article">
+
+                <h2>Target</h2>
+
+                  <h3>employment in the field of information technology</h3><br>
+
+                <h2>Work experience</h2>
+
+                  <h3>absent</h3><br>
+
+                <h2>Education</h2>
+
+                  <h3>higher, 2017 year, BSU, </h3>
+
+                  <h3>Faculty of Applied Mathematics and Computer Science</h3><br>
+
+                <h2>Skills</h2>
+
+                  <h3>C++, C#, JS, Java, SQL, UML</h3>
+
+            </div>
+
+            <div class="right-article">
+
+                <h2>Portfolio</h2>
+
+                  <ul>
+
+                    <li><a href="https://radiant-basin-45668.herokuapp.com/">Web application for education</a></li>
+
+                    <li><a href="https://github.com/Metalhammer4">GitHub</a></li>
+
+                  </ul><br>
+
+                <h2>Personal qualities</h2>
+
+                  <ul>
+
+                    <li>purposeful</li>
+
+                    <li>easily trained</li>
+
+                    <li>disciplined</li>
+
+                    <li>orderly</li>
+
+                    <li>precise</li>
+
+                    <li>hardworking</li>
+
+                    <li>diligent</li>
+                    
+                    <li>responsible</li>
+ 
+                    <li>communicable</li>
+
+                    <li>stress resistant</li>
+
+                    <li>without bad habbits</li>
+
+                  </ul>
+
+            </div>
+
+            <div class="clearfix"></div>
+
+          </div>
+
+        </div>
+
+      </div>
+
+        <div class="fixblock">
+    
+          <div class="inline">
+
+            <h2>Proposal</h2>
+
+              <form id="contact" action="#" method="post" name="contact">
+
+                <input class="txt" id="email" name="email" type="email" required title="Please fill out this field." placeholder="e-mail">
+ 
+                <textarea class="txtarea" id="msg" name="msg" required title="Please fill out this field." placeholder="message:"></textarea>
+
+                <button class="send">send</button>
+
+              </form>
+
+          </div>
+
+        </div>
+
+        <div class="footer">
+    
+          <div class="footer-columns">
+
+            <div class="footer-col">
+
+              <figure class="figcapt">
+
+                <img class="img-responsive" src="img/adr.png" alt="...">
+      
+                  <figcaption>
+
+                    <h5>Republic of Belarus, Minsk</h5>
+
+                  </figcaption>
+
+              </figure>
+
+            </div>
+
+            <div class="footer-col">
+
+              <figure class="figcapt">
+
+                <img class="img-responsive" src="img/tel.png" alt="...">
+    
+                  <figcaption>
+
+                    <h5>+375 17 244 34 50</h5>
```

Comment:
it's make sense to use `<a href` here.

Reed more: https://css-tricks.com/the-current-state-of-telephone-links/